source/Calamari.Tests/Fixtures/StructuredVariables/Approved/YamlVariableReplacerFixture.ShouldPreserveEncodingUtf8UnixNoBom.approved.txt

```diff
@@ -1,6 +1,6 @@
 64 61 67 67 65 72 3A 20   dagger: 
-E2 80 A0 0A 73 75 6E 6E   ???^sunn
-69 65 73 3A 20 22 5C 55   ies: "\U
-30 30 30 31 46 36 30 45   0001F60E
-22 0A 65 75 72 6F 3A 20   "^euro: 
-E2 82 AC 0A               ???^
\ No newline at end of file
+E2 80 A0 0D 0A 73 75 6E   ???^^sun

```

Comment:
I couldn't really get to the bottom of why the additional Carat characters are being added here.

I'm guessing this is a CRLF change https://stackoverflow.com/questions/13836352/what-is-the-utf-8-representation-of-end-of-line-in-text-file which could be an issue for Unix. I'm waiting to see what happens in the pipeline.

Reply:
It looks like the difference here and for other [tests](https://build.octopushq.com/buildConfiguration/OctopusDeploy_CalamariNetCore_PublishNugetPackagesToMyGet/11165366?buildTab=dependencies&mode=list&state=failed&type=snapshot&hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildTestsSection=true#11165308) is that it adds a carriage return. This looks to be a carriage return (0D) followed by null (00) followed by a line feed (0A)

![image](https://github.com/OctopusDeploy/Calamari/assets/101079287/b3c96f6f-d34b-4309-95eb-d6d489e1a82a)

Reply:
Possibly https://github.com/aaubry/YamlDotNet/issues/867